### PR TITLE
Notify HA of updates on fetch_data()

### PR DESCRIPTION
**Pull request by mkmer**
Need to tell listeners when new data is fetched - found when restarting the socket and calling fetch_data() HA was not notified of updated entities until an event message was received.

### CODE REVIEW

**Comment by abmantis** [comment]
I am not sure I understand the need for this change. `fetch_data()` is only called on startup when HA calls `connect()`, so there is no need to call the attr changed callback there.

**Comment by mkmer** [comment]
It is also called when the socket reconnects:
https://github.com/abmantis/whirlpool-sixth-sense/blob/f217ca550a275e1a04fd567a8efd092ad2968bce/whirlpool/appliance.py#L192C8-L192C8
which is mapped to this:
https://github.com/abmantis/whirlpool-sixth-sense/blob/f217ca550a275e1a04fd567a8efd092ad2968bce/whirlpool/eventsocket.py#L138

When the event socket reconnects, we don't "know" the states for sure, so we do a fetch to make sure everything is up to date.  Unfortunately, we don't tell HA that there is new data to update when this happens.

This will also allow us to drive 1 timed update function and still trigger all other entities (The bandaid for a crappy event socket)

File: whirlpool/eventsocket.py [hunk 1]
```diff
@@ -133,7 +133,7 @@ async def _run(self):
                             connected_msg_done = True
                             continue
 
-                        if not subscribe_msg_done:
```
Comment (mkmer) [inline whirlpool/eventsocket.py:136]: I forgot about this one - found it's not waiting for the connected_msg_done before sending the subscribe_msg_done.

Comment (abmantis) [reply whirlpool/eventsocket.py:136]: this elif is not needed, since the if block has `continue` already

**Comment by abmantis** [comment]
> It is also called when the socket reconnects: https://github.com/abmantis/whirlpool-sixth-sense/blob/f217ca550a275e1a04fd567a8efd092ad2968bce/whirlpool/appliance.py#L192C8-L192C8 which is mapped to this:
> 
> https://github.com/abmantis/whirlpool-sixth-sense/blob/f217ca550a275e1a04fd567a8efd092ad2968bce/whirlpool/eventsocket.py#L138
> 
> When the event socket reconnects, we don't "know" the states for sure, so we do a fetch to make sure everything is up to date. Unfortunately, we don't tell HA that there is new data to update when this happens.
> 
> This will also allow us to drive 1 timed update function and still trigger all other entities (The bandaid for a crappy event socket)

Ah I see! Makes sense!

File: whirlpool/eventsocket.py (hunk 1 above)
Comment (mkmer) [reply whirlpool/eventsocket.py]: True - I thought I had traced this through and it was behaving differently, but on second review it was working as expected.

**Review by abmantis — APPROVED** [review]

**Comment by abmantis** [comment]
Thanks!

**Comment by mkmer** [comment]
Can we get a release in pypi for this? Looking to activate the polling feature in HA and this is necessary for HA to update all entities.

**Comment by abmantis** [comment]
@mkmer sure! I just released it as v0.18.4.

**Pull request merged**